### PR TITLE
improve third party integration section

### PR DESCRIPTION
**Pull request by agallou**
- add some logos
- categorize the tools
- add some text between those
- add link on how to integrate those tools

Before : 
capture d ecran 2016-01-09 a 00 58 52

After: 
capture d ecran 2016-01-09 a 00 58 36

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -256,23 +256,68 @@ Note the global `$script` and `$runner` variables: reach any part of atoum to fi
...
+       <a href="https://jenkins-ci.org/">Jenkins</a>
+       <a href="http://docs.atoum.org/en/latest/cookbook.html#use-inside-jenkins-or-hudson" class="integration__doc-link">(see how)</a>
+    </li>
+    <li>
+       <img src="images/integrations_logos/circleci.png" />
+       <a href="https://circleci.com/">Circle CI</a>
+    </li>
 </ul>
+
+atoum has a **GREAT** integration with those IDEs:
```
Comment (Hywan) [inline features.md:277]: Seriously, **GREAT**!!!!!!!!

I guess `**great**` would have been enought :-).

**Comment by Hywan** [comment]
Too fast for the merge @Grummfy :-). Logos must be in SVG (see http://svgporn.com/). CSS is not correctly indented. Text is not correct too.

@Grummfy Can you revert the PR?

**Comment by jubianchi** [comment]
@Hywan we are not going to revert it does not make sense. We will fix what we can fix ;)

**Comment by agallou** [comment]
@Hywan : PR made to use svg (#15)
For CSS I haven't seen there were a logic in the css indent. just though there were intendation problems (not used to have indentation in css,only in scss/less). I we'll made a PR to fix that (could be usefull to indicate that in a CONTRIBUTING file).
Could you list what's not correct in the text ?

**Comment by Hywan** [comment]
@agallou I will try to make a PR to fix CSS. That's 4 spaces for a level of indentation, that's all. We can add a level of indentation if we are declaring properties for sub-components.

**Pull request merged**